server/app/App.ts

```diff
@@ -50,10 +74,7 @@ export class App {
         server.use(bodyParser.json());
         server.use(
           expressWinston.logger({
-            transports: [new winston.transports.Console()],
-            meta: false,
-            expressFormat: true,
-            statusLevels: true,
```

Comment:
Why did you remove these? I'm curious

I think it would be cool to see metadata from the Express logger logged to LogDNA when the backend is running in develop or main